cmd/kubeadm/app/util/pkiutil/pki_helpers_test.go

```diff
@@ -412,6 +433,103 @@ func TestTryLoadCertFromDisk(t *testing.T) {
 	}
 }
 
+func TestTryLoadCertChainFromDisk(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("Couldn't create tmpdir")
+	}
+	defer os.RemoveAll(tmpdir)
+
+	caCert, caKey, err := NewCertificateAuthority(&CertConfig{
+		Config: certutil.Config{CommonName: "Intermediate CA"},
+	})
+	if err != nil {
+		t.Errorf(
+			"failed to create intermediate CA cert and key with an error: %v",
+			err,
+		)
```

Comment:
this t.Errorf does not need to span on multiple lines.

Reply:
All requested formatting changes have been made in this commit: https://github.com/kubernetes/kubernetes/pull/97266/commits/d43e52497c602089b6d9f9809ca084f0625c695f